storage/tool/src/main/java/com/hortonworks/streamline/storage/tool/StorageProviderConfigurationReader.java

```diff
@@ -103,4 +108,24 @@ private static StorageProviderConfiguration readPhoenixProperties(Map<String, Ob
         return StorageProviderConfiguration.phoenix(jdbcDriverClass, jdbcUrl);
     }
 
+    /**
+     * storageProviderConfiguration:
+     *   providerClass: "com.hortonworks.streamline.storage.impl.jdbc.JdbcStorageManager"
+     *   properties:
+     *     db.type: "postgresql"
+     *     queryTimeoutInSecs: 30
+     *     db.properties:
+     *     dataSourceClassName: "org.postgresql.ds.PGSimpleDataSource"
+     *     dataSource.url: "jdbc:postgresql://localhost/test"
+     *     dataSource.user: "postgres"
+     *     dataSource.password: "postgres"
```

Comment:
Similar config was earlier there but was removed from the yaml file so I did not add it again.